OMCompiler/Compiler/BackEnd/BackendVariable.mo

```diff
@@ -1248,6 +1248,13 @@ algorithm
   end match;
 end isParam;
 
+public function makeParam
+  "Change variable to parameter"
+  input output BackendDAE.Var var;
+algorithm
+  var.varKind := BackendDAE.PARAM();
```

Comment:
Maybe restrict this to not overwrite `STATE()` and `STATE_DER()`? Fail for safety reasons?

Reply:
I don't think that either of them can end up in the globalKnownVars. I will try to make an example to proof myself wrong.